charts/cortx/README.md

```diff
@@ -113,6 +113,10 @@ helm uninstall cortx
 | externalKafka.adminUser | string | `"admin"` |  |
 | externalKafka.endpoints | list | `[]` |  |
 | fullnameOverride | string | `""` | A name that will fully override cortx.fullname |
+| global.cortx.image.pullPolicy | string | `""` | CORTX image pull policy. Overrides CORTX component `image.pullPolicy`. |
+| global.cortx.image.registry | string | `""` | CORTX Docker image registry. Overrides CORTX component `image.registry`. |
```

Comment:
Based on the general movement away from using Docker as the primary noun, we should use "container image", "container image registry" instead of "Docker image", "Docker registry", etc.